src/drf_yasg/codecs.py

```diff
@@ -176,7 +176,14 @@ def represent_odict(self, mapping, flow_style=None):  # pragma: no cover
                 node.flow_style = best_style
         return node
 
+    def represent_text(self, text):
+        if len(text.splitlines()) > 1:
```

Comment:
It would likely be more efficient to use `'\n' in text or '\r' in text` here. Serializing to YAML is already very slow so let's not make it worse than needed.

Reply:
Done